vendor/modules.txt

```diff
@@ -360,15 +360,10 @@ github.com/pmezard/go-difflib/difflib
 ## explicit; go 1.16
 github.com/pquerna/cachecontrol
 github.com/pquerna/cachecontrol/cacheobject
-<<<<<<< HEAD
```

Comment:
I believe this change is related to the last commit. Can we fixup this commit to keep the history clean?

Reply:
Fixed.